### ISSUE TITLE
[Bug]: Sustainability subscribers broke a preview posting

### ISSUE THREAD

**Issue by vlkov** [microsoft/ALAppExtensions#27706]
### Describe the issue

Hello,
We have the next error on document preview posting.
Error message: 
The binding of codeunit 6228 was unsuccessful. The codeunit has already been bound.

AL call stack: 
"Sust. Purchase Subscriber"(CodeUnit 6225).OnAfterBindSubscription line 3 - Sustainability by Microsoft
"Gen. Jnl.-Post Preview"(CodeUnit 19).OnAfterBindSubscription(Event) line 2 - Base Application by Microsoft
"Gen. Jnl.-Post Preview"(CodeUnit 19).PreviewStart line 16 - Base Application by Microsoft
"Gen. Jnl.-Post Preview"(CodeUnit 19).Preview line 7 - Base Application by Microsoft

The issue related to extension Sustainability. This app added some tables to preview but it was not implemented correctly. When some error occurs in the preview posting the manual subscription codeunit stays already bound. So on the next run of preview posting, this error occurs.
I suggest fixing it and make like in codeunit 6496 "Serv. Posting Preview Binding" from the Base Application.

Basically, we need to use bindSubscription inside a if then and add a clear temporary record from the previous possible run.

### Expected behavior

There is no error on the preview posting related to the already bounded codeunit 6228 

### Steps to reproduce

1. Prepare a document that will cause a posting error.
2. Run Preview posting.
3. Run preview posting again

### I will provide a fix for a bug

- [x] I will provide a fix for a bug

**Comment by JesperSchulz** [comment]
@vlkov, this very much sounds like a case for support. Would you still want to provide a fix? We can process this one quickly and I could backport to the next minor release (if we hurry).

**Comment by vlkov** [comment]
@JesperSchulz  Yes, pull request created
